CRITICAL - 2025-08-05 14:16:20 --> TypeError: App\Controllers\Home::agregarEmplado(): Return value must be of type string, none returned
[Method: POST, Route: agregarEmpleado]
in APPPATH\Controllers\Home.php on line 45.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-05 14:17:38 --> mysqli_sql_exception: Duplicate entry '258' for key 'PRIMARY' in C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `em...', 0)
#1 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\app\Controllers\Home.php(43): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
#9 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-08-05 14:17:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '258' for key 'PRIMARY'
[Method: POST, Route: agregarEmpleado]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (:empleado_id:, :nombre:, :apellido:, :telefono:, :puesto_id:, :fecha_nac:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Home.php(43): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-05 14:17:38 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '258' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'258\', \'Pato\', \'Quaso\', \'25280000\', \'7\', \'2025-08-02\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'258\', \'Pato\', \'Quaso\', \'25280000\', \'7\', \'2025-08-02\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (:empleado_id:, :nombre:, :apellido:, :telefono:, :puesto_id:, :fecha_nac:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Home.php(43): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-05 14:17:38 --> [Caused by] mysqli_sql_exception: Duplicate entry '258' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'258\', \'Pato\', \'Quaso\', \'25280000\', \'7\', \'2025-08-02\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'258\', \'Pato\', \'Quaso\', \'25280000\', \'7\', \'2025-08-02\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'258\', \'Pato\', \'Quaso\', \'25280000\', \'7\', \'2025-08-02\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (:empleado_id:, :nombre:, :apellido:, :telefono:, :puesto_id:, :fecha_nac:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Home.php(43): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-05 14:18:06 --> mysqli_sql_exception: Duplicate entry '258' for key 'PRIMARY' in C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `em...', 0)
#1 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\app\Controllers\Home.php(43): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
#9 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-08-05 14:18:06 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '258' for key 'PRIMARY'
[Method: POST, Route: agregarEmpleado]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (:empleado_id:, :nombre:, :apellido:, :telefono:, :puesto_id:, :fecha_nac:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Home.php(43): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-05 14:18:06 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '258' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'258\', \'floo\', \'fernan\', \'25280000\', \'7\', \'2025-08-02\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'258\', \'floo\', \'fernan\', \'25280000\', \'7\', \'2025-08-02\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (:empleado_id:, :nombre:, :apellido:, :telefono:, :puesto_id:, :fecha_nac:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Home.php(43): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-05 14:18:06 --> [Caused by] mysqli_sql_exception: Duplicate entry '258' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'258\', \'floo\', \'fernan\', \'25280000\', \'7\', \'2025-08-02\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'258\', \'floo\', \'fernan\', \'25280000\', \'7\', \'2025-08-02\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'258\', \'floo\', \'fernan\', \'25280000\', \'7\', \'2025-08-02\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (:empleado_id:, :nombre:, :apellido:, :telefono:, :puesto_id:, :fecha_nac:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Home.php(43): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-05 14:21:57 --> ParseError: syntax error, unexpected variable "$empleados", expecting "," or ";"
[Method: POST, Route: agregarEmpleado]
in APPPATH\Controllers\Home.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\fs2025\\codeigniter4-Blackwave\\codeigniter4-framework-dbc07dc\\app\\Controllers\\Home.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Home')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Home')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\Home', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-05 14:23:09 --> mysqli_sql_exception: Duplicate entry '858' for key 'PRIMARY' in C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `em...', 0)
#1 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\app\Controllers\Home.php(44): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
#9 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-08-05 14:23:09 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '858' for key 'PRIMARY'
[Method: POST, Route: agregarEmpleado]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (:empleado_id:, :nombre:, :apellido:, :telefono:, :puesto_id:, :fecha_nac:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Home.php(44): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-05 14:23:09 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '858' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'858\', \'gsgsg\', \'sgesgd\', \'78784856\', \'7\', \'2025-08-13\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'858\', \'gsgsg\', \'sgesgd\', \'78784856\', \'7\', \'2025-08-13\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (:empleado_id:, :nombre:, :apellido:, :telefono:, :puesto_id:, :fecha_nac:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Home.php(44): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-05 14:23:09 --> [Caused by] mysqli_sql_exception: Duplicate entry '858' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'858\', \'gsgsg\', \'sgesgd\', \'78784856\', \'7\', \'2025-08-13\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'858\', \'gsgsg\', \'sgesgd\', \'78784856\', \'7\', \'2025-08-13\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'858\', \'gsgsg\', \'sgesgd\', \'78784856\', \'7\', \'2025-08-13\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (:empleado_id:, :nombre:, :apellido:, :telefono:, :puesto_id:, :fecha_nac:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Home.php(44): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-05 14:44:13 --> mysqli_sql_exception: Duplicate entry '859' for key 'PRIMARY' in C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `em...', 0)
#1 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `em...')
#2 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `em...')
#3 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `em...', Array, false)
#4 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\BaseModel.php(838): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Model.php(797): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\app\Controllers\Home.php(44): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
#9 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2025\codeigniter4-Blackwave\codeigniter4-framework-dbc07dc\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-08-05 14:44:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '859' for key 'PRIMARY'
[Method: POST, Route: agregarEmpleado]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (:empleado_id:, :nombre:, :apellido:, :telefono:, :puesto_id:, :fecha_nac:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Home.php(44): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-05 14:44:13 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '859' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'859\', \'gsgsg\', \'sgesgd\', \'78784856\', \'7\', \'2025-08-13\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'859\', \'gsgsg\', \'sgesgd\', \'78784856\', \'7\', \'2025-08-13\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (:empleado_id:, :nombre:, :apellido:, :telefono:, :puesto_id:, :fecha_nac:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Home.php(44): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-05 14:44:13 --> [Caused by] mysqli_sql_exception: Duplicate entry '859' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'859\', \'gsgsg\', \'sgesgd\', \'78784856\', \'7\', \'2025-08-13\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'859\', \'gsgsg\', \'sgesgd\', \'78784856\', \'7\', \'2025-08-13\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (\'859\', \'gsgsg\', \'sgesgd\', \'78784856\', \'7\', \'2025-08-13\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `emplados` (`empleado_id`, `nombre`, `apellido`, `telefono`, `puesto_id`, `fecha_nac`) VALUES (:empleado_id:, :nombre:, :apellido:, :telefono:, :puesto_id:, :fecha_nac:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(838): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(797): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Home.php(44): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->agregarEmplado()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
